nethserver-cockpit.spec

```diff
@@ -40,6 +40,7 @@ perl createlinks
 
 %install
 (cd root ; find . -depth -not -name '*.orig' -print | cpio -dump %{buildroot})
+mkdir -p %{buildroot}/etc/nethserver/authorizations/
```

Comment:
I'd like to avoid a new directory... Let's put the pam_listfile in `/etc/nethserver/cockpit.allow` or similar.

Reply:
done